TKGame/BackEnd/EntityManager.cs

```diff
@@ -8,12 +8,19 @@
 using System.Threading.Tasks;
 using TKGame.Components.Interface;
 using TKGame.Level_Editor_Content;
+using TKGame.Players.Components;
 
 namespace TKGame.BackEnd
 {
     // Might change from static
     static class EntityManager
     {
+
+# region components
+        public static C_Player_WeaponAttack weaponAttack = new C_Player_WeaponAttack(); //Only for use with player but needs to be in the EntityManagers Update functions.
```

Comment:
why does it need to be there?

Reply:
needs access to the entities list from entitymanager to get the graphical changes working.